cortex/kernel_features/llm_device/llm_device.py

```
@@ -0,0 +1,573 @@
+        self.model = "claude-sonnet-4-20250514"
+
+        if self.api_key:
+            try:
+                import anthropic
+                self.client = anthropic.Anthropic(api_key=self.api_key)
+            except ImportError:
+                self.client = None
+        else:
+            self.client = None
+
+    def complete(self, prompt: str, config: dict = None) -> str:
```

Ruff (0.14.7)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)